utils/utils.go

```diff
@@ -0,0 +1,27 @@
+// Package utils provides utility functions for blueprint path construction
```

Comment:
`utils/utils.go` filename sounds weird. Can we change the filename to something like - `utils/path.go` or if you can think of any better name.